finance-api/api-reference.md

```diff
@@ -0,0 +1,288 @@
+|totalBalance |integer |The account balance in the currency's smallest unit. It can be negative, such as when refunds are greater than sales. In the following example, the account balance is -3 SEK. <br/><pre>{ <br/>totalBalance: -300, <br/>currencyId: "SEK" <br/>}</pre> 
+|nextPayoutAmount |integer |The amount of money to be paid out to the merchant.  
+|discountRemaining |integer |The amount of discounts that remains in merchant's vouchers. The vouchers are offered by Zettle.<br/>For example, a merchant has a voucher worthy 100 SEK from a Zettle marketing campaign. For a transaction of 200 SEK, Zettle will subtract two SEK for the transaction fee from the voucher. Then the merchant will have a remaining discount of 98 SEK. 
+|periodicity |string |The period between each payout that is set by the merchant . It can be `DAILY`, `WEEKLY`, and `MONTHLY`.   
+</details>
+
+
+## Examples
+
+### Fetch current balance for a liquid account
+In the following example, as `at` is not specified, the current balance is fetched by default.
+
+Request
+```
+GET /organizations/self/accounts/liquid/balance
+```
+Response
+```json
+{
+    "data": {
+        "totalBalance": 195,
+        "currencyId": "GBP"
+    }
+}
+```
+
+### Fetch transactions for a liquid account
+The following example fetches transactions from the merchant's Zettle liquid account from 1 January, 2020 to 31 December, 2020. With pagination set by `limit` and `offset`, three transactions are returned at a time until all transactions are fetched during that specific period.
+
+Request
+```
+GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-12-31&limit=3&offset=0
+```
+```
+GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-12-31&limit=3&offset=3
+```
+
+Response
+```json
+{
+    "data": [
+        {
+            "timestamp": "2020-11-21T04:00:15.704+0000",
+            "amount": -621,
+            "originatorTransactionType": "PAYMENT_FEE",
+            "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+        },
+        {
+            "timestamp": "2020-11-21T04:00:15.697+0000",
+            "amount": 1100,
+            "originatorTransactionType": "PAYMENT",
+            "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+        },
+        {
+            "timestamp": "2020-09-10T09:51:35.162+0000",
+            "amount": 5925,
+            "originatorTransactionType": "FAILED_PAYOUT",
+            "originatingTransactionUuid": "d8550d7a-f347-11ea-9612-3bce5300b9a9"
+        }
+       ...
+    ]
+}
+```
+
+### Fetch payout information on a specific day
+The following example fetches all payout information from the merchant's Zettle liquid account on 7 June, 2021.
+
+Request
+```
+GET /organizations/self/payout-info?at=2021-06-07
+```
+Response
+```json
+{
+    "data": {
+        "totalBalance": 195,
+        "currencyId": "GBP",
+        "nextPayoutAmount": 0,
```

Comment:
The example is weird with `nextPayoutAmount=0 ` when `totalBalance=195`.

I think more realistic, happy flow example should be used.